src/nomad_catalysis/example_uploads/__init__.py

```diff
@@ -0,0 +1,8 @@
+from nomad.config.models.plugins import ExampleUploadEntryPoint
+
+catalysis = ExampleUploadEntryPoint(
+    title='Heterogeneous Catalysis Example',
+    category='Examples',
```

Comment:
Which is the better category? There is also a category 'FAIRmat examples', probably better?

Reply:
I think "FAIRmat examples" is something where Area B pushes there NeXus things but I think it would be good if we add something there!
```suggestion
    category='FAIRmat examples',
```